### PR TITLE
Feature/dev guduodata

### PR DESCRIPTION
**Pull request by Gem1ni**
## 该 PR 相关 Issue / Involved issue

Close #

## 完整路由地址 / Example for the proposed route(s)

```routes
/guduodata/daily
```

## 新RSS检查列表 / New RSS Script Checklist
  
- [x] New Route
- [x] Documentation
  - [x] CN
  - [ ] EN
- [x] 全文获取 fulltext
  - [x] Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] 日期和时间 date and time
  - [ ] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added 
- [ ] `Puppeteer`

## 说明 / Note

还处理了 goverment.md 以及 bbs.md 部分 Route 标签未正确关闭的问题

### CODE REVIEW

File: lib/v2/guduodata/radar.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'guduodata.com': {
+        _name: '骨朵数据',
+        data: [
+            {
+                title: '日榜',
+                docs: 'https://docs.rsshub.app/other.html#gu-duo-shu-ju',
+                source: ['/'],
+                target: '/daily',
```
Comment (TonyRL) [inline lib/v2/guduodata/radar.js:9]:
```suggestion
                target: '/guduodata/daily',
```

File: docs/government.md [hunk 2]
```diff
@@ -238,7 +238,7 @@ pageClass: routes
 
 ### 新闻公布
 
-<Route author="linbuxiao" example="/icac/news/sc" path="/icac/news/:lang?" :paramsDesc="['语言，留空为`sc`，支持`sc`（简中），`tc`（繁中），`en`（英文）']">
+<Route author="linbuxiao" example="/icac/news/sc" path="/icac/news/:lang?" :paramsDesc="['语言，留空为`sc`，支持`sc`（简中），`tc`（繁中），`en`（英文）']" />
```
Comment (TonyRL) [inline docs/government.md:241]: Route author already submitted changes for line in https://github.com/DIYgod/RSSHub/pull/8273

File: docs/other.md [hunk 3]
```diff
@@ -405,6 +405,14 @@ type 为 all 时，category 参数不支持 cost 和 free
 | ---- | ---- | ----- |
 | yi   | zhu  | shang |
 
+</Route>
```
Comment (TonyRL) [inline docs/other.md:408]: You forgot line 400
`<Route author="LogicJake" example="/gushiwen/recommend/zhushang" path="/gushiwen/recommend/:annotation?"> `

Comment (Gem1ni) [reply docs/other.md:408]: Not me, @LogicJake forgot line 400

File: docs/government.md (hunk 2 above)
Comment (Gem1ni) [reply docs/government.md:241]: ok

**Pull request merged**